diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+_Users/grace/wdi/try._
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+_An online resource that allows you to share, update, different versions of a project and to keep track of those versions._
 
```

Comment:
Correct.

It is a web client UI for git, git is the version control, github is a ui for using it.